app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -50,32 +50,32 @@ class ContinuePlayingFragmentPresenter @Inject constructor(
     return binding.root
   }
 
-  private fun createRecyclerViewAdapter(): BindableAdapter<ContinuePlayingItemViewModel> {
+  private fun createRecyclerViewAdapter(): BindableAdapter<RecentlyPlayedItemViewModel> {
     return BindableAdapter.MultiTypeBuilder
-      .newBuilder<ContinuePlayingItemViewModel, ContinuePlayingItemViewModel.ViewType> { viewModel ->
+      .newBuilder<RecentlyPlayedItemViewModel, RecentlyPlayedItemViewModel.ViewType> { viewModel ->
         when (viewModel) {
-          is SectionTitleViewModel -> ContinuePlayingItemViewModel.ViewType.VIEW_TYPE_SECTION_TITLE_TEXT
-          is OngoingStoryViewModel -> ContinuePlayingItemViewModel.ViewType.VIEW_TYPE_SECTION_STORY_ITEM
+          is SectionTitleViewModel -> RecentlyPlayedItemViewModel.ViewType.VIEW_TYPE_SECTION_TITLE_TEXT
+          is OngoingStoryViewModel -> RecentlyPlayedItemViewModel.ViewType.VIEW_TYPE_SECTION_STORY_ITEM
           else -> throw IllegalArgumentException("Encountered unexpected view model: $viewModel")
         }
       }
       .registerViewDataBinder(
-        viewType = ContinuePlayingItemViewModel.ViewType.VIEW_TYPE_SECTION_TITLE_TEXT,
+        viewType = RecentlyPlayedItemViewModel.ViewType.VIEW_TYPE_SECTION_TITLE_TEXT,
         inflateDataBinding = SectionTitleBinding::inflate,
         setViewModel = SectionTitleBinding::setViewModel,
         transformViewModel = { it as SectionTitleViewModel }
       )
       .registerViewDataBinder(
-        viewType = ContinuePlayingItemViewModel.ViewType.VIEW_TYPE_SECTION_STORY_ITEM,
+        viewType = RecentlyPlayedItemViewModel.ViewType.VIEW_TYPE_SECTION_STORY_ITEM,
         inflateDataBinding = OngoingStoryCardBinding::inflate,
         setViewModel = OngoingStoryCardBinding::setViewModel,
         transformViewModel = { it as OngoingStoryViewModel }
       )
       .build()
   }
 
-  private fun getContinuePlayModel(): ContinuePlayViewModel {
-    return viewModelProvider.getForFragment(fragment, ContinuePlayViewModel::class.java)
+  private fun getContinuePlayModel(): RecentlyPlayedViewModel {
```

Comment:
Ditto here change this method to getRecentlyPlayedModel()

Reply:
Done